### PR TITLE
Wrap too long push mirror addresses

### PR DESCRIPTION
**Pull request by JakobDev**
I noticed that it is possible to break the push mirror list by entering a too long URL. This should fix it.

| Before         | After     | 
|--------------|-----------|
|Before|After|

### CODE REVIEW

File: templates/repo/settings/options.tmpl [hunk 1]
```diff
@@ -169,7 +169,11 @@
 						{{range .PushMirrors}}
 						<tr>
 							{{$address := MirrorRemoteAddress $.Context $.Repository .GetRemoteName true}}
-							<td>{{$address.Address}}</td>
+							{{if gt (len  $address.Address) 50}}
+								<td title="{{$address.Address}}">{{slice $address.Address 0 50}}...</td>
```
Comment (delvh) [inline templates/repo/settings/options.tmpl]: Hmm, wouldn't it be better to always send the whole address and letting CSS take care of it with overflow?

Comment (JakobDev) [reply templates/repo/settings/options.tmpl]: I'm not good as CSS. I know that something like `text-overflow: ellipsis` exists, but I can't get it to look good., so I need a little help with this.

**Comment by wxiaoguang** [comment]
IMO work-breaking could be better for such case.

If you truncate the address, you won't be able to read it / copy it fully.

With work-breaking, you get a multiple-line field on the UI and can see the full address.

**Comment by wxiaoguang** [comment]
#21251 could be an example, it fixes the long texts in container table.

File: templates/repo/settings/options.tmpl (hunk 1 above)
Comment (wxiaoguang) [reply templates/repo/settings/options.tmpl]: Now we have CSS helpers, it just needs `class="gt-word-break"` on the `<td>`

image

**Review by wxiaoguang — APPROVED** [review]

**Review by delvh — APPROVED** [review]

**Pull request merged**